pkg/systemd/quadlet/podmancmdline.go

```diff
@@ -3,8 +3,17 @@ package quadlet
 import (
 	"fmt"
 	"sort"
+	"os"
 )
 
+func PodmanBinary() string {
+     podman := os.Getenv("PODMAN")
+     if len(podman) > 0 {
+          return podman
+     }
+     return "/usr/bin/podman"
```

Comment:
I have the feeling this must be configurable via built time option, i.e. it should use `BINDIR` from the makefile.